modules/rest_api/controllers/services/rest.php

```diff
@@ -1409,6 +1406,23 @@ private function getEsPostData($postObj, $format, $file, $isSearch) {
           $fields[] = 'identification';
           $fields[] = 'occurrence.zero_abundance';
         }
+        elseif (preg_match('/^#determiner(.*)#$/', $field)) {
+          $fields[] = 'event.recorded_by';
+          $fields[] = 'identification.identified_by';
+        }
+        elseif (preg_match('/^#sitename(.*)#$/', $field)) {
+          $fields[] = 'location.verbatim_locality';
+        }
+        elseif (preg_match('/^#method(.*)#$/', $field)) {
+          $fields[] = 'event.sampling_protocol';
+        }
+        elseif (preg_match('/^#sample_occurrence_comment(.*)#$/', $field)) {
+          $fields[] = 'event.event_remarks';
+          $fields[] = 'occurrence.occurrence_remarks';
+          $fields[] = '_id';
+          $fields[] = 'id';
+          $fields[] = 'occurrence.id';
```

Comment:
The field containing the occurrence ID is just called 'id'. There is also a field called '_id' which is the one with the unique prefix specifying which warehouse the record came from (in case we load several warehouses into 1 ES instance).